src/main/scala/housekeeper/Lambda.scala

```diff
@@ -28,7 +28,7 @@ object Lambda extends Logging {
     if (bounce.isPermanent) {
       val deletionF = Future.traverse(bounceNotification.bouncedAddresses)(alertDeletion.deleteAllAlertsForEmailAddress)
       val notificationF = alertDevsIfEmailWasSuppressed(bounceNotification)
-      for { _ <- deletionF; _ <- notificationF } yield ()
+      for { d <- deletionF; _ <- notificationF } yield ()
```

Comment:
This change is unnecessary I think!

```suggestion
      for { _ <- deletionF; _ <- notificationF } yield ()
```

Reply:
I did change it to how it was 👍🏼